app/client/components/billing/invoicesTable.tsx

```diff
@@ -293,7 +301,20 @@ export const InvoicesTable = (props: InvoicesTableProps) => {
                         )} ${tableRow.currencyISO}`}
                   </div>
                   <div css={tdCss2(index)}>
-                    <a css={invoiceLinkCss} href={tableRow.pdfPath}>
+                    <a
+                      css={invoiceLinkCss}
+                      href={tableRow.pdfPath}
+                      onClick={() => {
+                        return trackEvent({
+                          eventCategory: "invoice",
+                          eventAction: "click",
+                          eventLabel: `view_${tableRow.product?.replace(
+                            /\s+/g,
+                            ""
+                          )}_pdf_invoice`
```

Comment:
instead of doing a replace here, you could use `urlPart` of the `ProductType` (or rather `specificProductType`) - which would be more consistent with other GA events